src/Messaging/Jobs/AppUninstalledJob.php

```diff
@@ -71,6 +71,9 @@ public function handle(
 
         // Get the shop
         $shop = $shopQuery->getByDomain($this->domain);
+        if ( !$shop ) {
+            return true;
```

Comment:
`handle(...)` returns a `true`. It's `parent::handle(...)` also returns a `true`. The linter/IDE yells at us if we return without specifying a boolean. I think it might be cleaner to return `true` here.